src/pages/components/Navbar/DeveloperNavbar.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Dialog } from '@headlessui/react';
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline';
+
+const navigation = [
+  { name: 'Features', href: '#' },
+  { name: 'Try it Out', href: '#' },
+  { name: 'About', href: '#' },
+  { name: 'Register API Key', href: '#' },
+  { name: 'Docs', href: '#' },
+];
+
+export default function NewHeader() {
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
+
+  return (
+    <div className="bg-white">
```

Comment:
we use ChakraUI for this project - please replace all div, a, h1, etc. with their ChakraUI equivalents (Box, Link, Heading, etc.)